central/telemetry/centralclient/interceptors.go

```diff
@@ -0,0 +1,107 @@
+package centralclient
+
+import (
+	"strings"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/set"
+	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+)
+
+var (
+	trackedPaths set.FrozenSet[string]
```

Comment:
I find it hard to reason about the lifetime, ownership, and initialization of `trackedPaths`. It is initialized in `getInstanceConfig()` which is called once as part of `phonehome.Config` singleton initialization. So there is an implicit assumption that `getInstanceConfig()` can (or should) only be called once. 

I'd suggest to make it part of `Config` and pass into interceptors from the `Config` singleton.

Reply:
`trackedPath` is a central specific feature, so it shouldn't be part of `phonehome.Config`. 
`getInstanceConfig()` is a central telemetry configuration initialization function, which initializes this central specific package level variable.